.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,30 @@
+## **SUBMISSIONS LACKING A RESPONSE TO ONE OF THE FIELDS WILL BE DELETED**
+
+### Issue Description:
+<!--- Use our forum https://forums.pocketmine.net for questions -->
+
+> Write a short description about the issue
+
+### Steps to reproduce the issue: 
+
+> How did you come across the issue? Did you enter a command? Did you test a plugin? Help us locate the problem by adding steps to reproduce the issue!
+
+### Operating System and Version
+<!--- Use the /version command to recieve these details. 
+PocketMine version should include the Jenkins build number or the git commit hash.
+--->
+> PocketMine-MP Version:
+PHP Version:
+Server Operating System:
+Game Version- PE or Win10: 
+
+### Crash Dump, Backtrace, or any Other Information
+
+> Please use gist or https://www.filedropper.com/ and add the link here
+
+### Make sure all the checkboxes below are checked. If not, the issue will be deleted.
+<!--- The checkboxes can be checked AFTER the issue is submitted. -->
+- [ ] This issue isn't duplicated - check by using the search bar at the top of your screen, and selecting the "issues" tab.
+- [ ] This issue is written in english for clarity. Use https://translate.google.com to translate to english.
+- [ ] This issue does not use abbreviations- type out the full word or phrase please. (Understandability)
+- [ ] The cause of this issue is not because of a plugin
```

Comment:
And an issue can be related to many things, not just core bugs. There are professional issues that are not related to the game at all.